tests/acceptance/AdminDatasetForm.feature

```diff
@@ -131,10 +131,15 @@ Feature: form to update dataset details
   Scenario: Can create/reset private url
     When I am on "/adminDataset/update/id/5"
     And I press the button "Create/Reset Private URL"
-    And I wait "1" seconds
+    And I wait "3" seconds
     Then I should see current url contains "/dataset/100039/token/"
     And I should see "Genomic data of the Puerto Rican Parrot (Amazona vittata) from a locally funded project."
 
+  @ok
+  Scenario: Can't see create/reset private url
+    When I am on "/adminDataset/update/id/8"
+    Then I should not see "Create/Reset Private URL"
+
```

Comment:
**question:** This scenario is also passing when I executed it from the develop branch, so I was wondering what is the purpose of this test?

Reply:
it's just a rearrangement of tests. Since dataset 8 is already published, we might as well reuse it directly.